projects/cellana/index.js

```diff
@@ -9,6 +11,13 @@ async function _getPools() {
   if (!poolsCache) poolsCache = function_view({ functionStr: "0x4bf51972879e3b95c4781a5cdcb9e1ee24ef483e7d22f2d903626f126df62bd1::liquidity_pool::all_pool_addresses", type_arguments: [], args: [] })
   return poolsCache
 }
+async function _getCoinGeckoId() { 
+  if (!cgkIdCache) {
+    cgkIdCache =await axios.get("https://api-v2.cellana.finance/api/v1/tool/cgk-id").then((res) => res.data)
```

Comment:
@waynebruce0x can you instead add an adapter to add these mappings to coins server?

Reply:
how can i do this mate

Reply:
> how can i do this mate

some one my partner  not apply there token into DefiLlama, I don't know what to do

Reply:
Because some tokens listed on Cellana and Coingecko do not exist in the coin server, it causes Defillama's calculations to miss these assets.